src/sentry/rules/filters/latest_adopted_release_filter.py

```diff
@@ -28,57 +29,58 @@
 
 
 class LatestAdoptedReleaseForm(forms.Form):
-    release_age_type = forms.ChoiceField(choices=list(model_age_choices))
-    age_comparison = forms.ChoiceField(choices=list(age_comparison_choices))
-    environment_id = forms.CharField(required=False)
+    oldest_or_newest = forms.ChoiceField(choices=list(model_age_choices))
+    older_or_newer = forms.ChoiceField(choices=list(age_comparison_choices))
+    environment = forms.CharField(required=False)
```

Comment:
Not a blocker: having env as a ChoiceField pre-populated with org's valid envs would be nice.

Reply:
Not practical unfortunately - there are orgs with 100ks of envs